### PR TITLE
Bug 1943649: Add command for network-check-target instead of hello-openshift

### PR DESCRIPTION
**Pull request by yselkowitz**

### CODE REVIEW

**Comment by smarterclayton** [comment]
Thank you - @openshift/networking if you can review so we can break the reference to hello-openshift which is being replaced with specific  use case solutions

File: cmd/cluster-network-check-target/main.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	"net"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+func checktargetHandler(w http.ResponseWriter, r *http.Request) {
+	response := os.Getenv("RESPONSE")
+	if len(response) == 0 {
+		response = "Hello OpenShift!"
```
Comment (danwinship) [inline cmd/cluster-network-check-target/main.go:14]: we should improve this later but we can get it in as-is now

File: cmd/cluster-network-check-target/main.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	}
+
+	// Echo back the port the request was received on
+	// via a "request-port" header.
+	addr := r.Context().Value(http.LocalAddrContextKey).(net.Addr)
+	if tcpAddr, ok := addr.(*net.TCPAddr); ok {
+		w.Header().Set("x-request-port", strconv.Itoa(tcpAddr.Port))
+	}
```
Comment (danwinship) [inline cmd/cluster-network-check-target/main.go:22]: (I presume this was added for network-edge's purposes; we don't need this.)

**Review by danwinship — APPROVED** [review]
/lgtm

File: cmd/cluster-network-check-target/main.go (hunk 2 above)
Comment (yselkowitz) [reply cmd/cluster-network-check-target/main.go:22]: Now that you mention it, you're right: https://github.com/openshift/origin/commit/938a8e99bcbd0d0ca33bf827dfd209859cf525be

**Comment by danwinship** [comment]
/override ci/prow/e2e-gcp-ovn
that job is very red right now but it's irrelevant to this PR

**Pull request merged**